core/translate.c

```diff
@@ -1438,7 +1438,7 @@ record_translation_info(dcontext_t *dcontext, fragment_t *f, instrlist_t *existi
      * final entry for the inserted jmp, have recreate_ know about it and cut
      * in half the typical storage reqts.
      */
-#   define NUM_INITIAL_TRANSLATIONS 2
```

Comment:
This style is different: it's lining up the define with the indentation of the code around it to make it more readable.  I kind of like this for defines inside function bodies, though I'm not sure how to reconcile it with an automated style rule that doesn't have a full parser.  WDYT?